go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/btcsuite/btcd v0.22.0-beta // indirect
 	github.com/cespare/xxhash v1.1.0 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/coinbase/rosetta-sdk-go v0.6.10 // indirect
+	github.com/coinbase/rosetta-sdk-go v0.7.0 // indirect
```

Comment:
latest is `v0.7.3`, should we be using this?

Reply:
guess it's good to stay in line with this release (altho patches should'nt matter) https://github.com/cosmos/cosmos-sdk/commit/a8071f84d3e50558cdc447c27688624e83542552